### PR TITLE
fix: build module before publishing in CI

### PR DESCRIPTION
**Pull request by vladfrangu**

### CODE REVIEW

**Review by B4nan — APPROVED** [review]
btw that's what the `prepare` script was for, as it trigger the build after install

**Comment by vladfrangu** [comment]
> btw that's what the `prepare` script was for, as it trigger the build after install

Yeah.. some repositories moved to prepublishOnly (as to not slow down installs), but I see in this repo the prepublishOnly script is a bash one (and I'll need to sharpen my bash knowledge/write a quick js script for crossplatform compat to solve it). Let me know what's ideal 👍

**Comment by B4nan** [comment]
i am fine with how it is now, the build should be done explicitly in CI as it is and the `prepublishOnly` do not have to be crossplatform ready, as publishing is only meant to be executed in CI where we control the environment

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "apify-client",
-	"version": "2.0.0",
```
Comment (mnmkng) [inline package.json:3]: No need to bump version. We have not released 2.0 yet.

**Review by mnmkng — APPROVED** [review]

**Pull request merged**